packages/wdio-config/src/lib/ConfigParser.js

```diff
@@ -256,6 +262,16 @@ export default class ConfigParser {
         return this._capabilities
     }
 
+    static removeLineNumbers(pattern) {
+        if (pattern.includes(':')) {
+            pattern = pattern.split(':')[0]
+        }
+        return pattern
+    }
+
+    static isCucumberFeatureWithLineNumber(patterns) {
+        return patterns.find((pattern) => pattern.includes(':'))
+    }
```

Comment:
These two methods seem to be utility methods. Can you move them to the utils file?

Reply:
I will do when I get into the office today! :-)

Reply:
Feedback addressed